### DISCUSSION TITLE
Confirm dialog covering text

### DISCUSSION THREAD

**Discussion by bojohnson5** [folke/noice.nvim#450] [Q&A]
I've found if I'm working in a smaller terminal and do a search and replace, the confirm dialog can cover text I might want to replace. In order to turn off the confirm dialog while doing search and replace, can I use a route to send that particular confirm dialog to the statusline, or would I turn off the confirm dialog all together? If turning off all together, what's the best way to do that?

**Comment by jborbik** [comment]
I wanted to double on this. Another solution would be the possibility to scroll the buffer up or down if the confirmation dialog is covering the text. Could we make that possible?
